backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/Wav.java

```diff
@@ -169,5 +157,36 @@ public int read (byte[] buffer) throws IOException {
 			} while (offset < buffer.length);
 			return offset;
 		}
+
+		/** List is a combination of Audacity's export formats and Windows ACM. For a more thorough list, see
+		 * https://wiki.multimedia.cx/index.php/TwoCC
+		 * @param type 16-bit value from the fmt chunk.
+		 * @return A human-readable name for the codec. */
+		private String getCodecName (int type) {
+			switch (type) {
+			case 0x0002:
+				return "Microsoft ADPCM";
+			case 0x0006:
+				return "ITU-T G.711 A-law";
+			case 0x0007:
+				return "ITU-T G.711 u-law";
+			case 0x0011:
+				return "IMA ADPCM";
+			case 0x0022:
+				return "DSP Group TrueSpeech";
+			case 0x0031:
+				return "Microsoft GSM 6.10";
+			case 0x0040:
+				return "Antex G.721 ADPCM";
+			case 0x0070:
+				return "Lernout & Hauspie CELP 4.8kbps";
+			case 0x0072:
+				return "Lernout & Hauspie CBS 12kbps";
+			case 0xfffe:
+				return "Extensible";
+			default:
+				return "Unknown";
+			}
+		}
```

Comment:
If you dislike what the formatter does to switches, you can use `// @off` and `// @on` (or at least you used to be able to).